GreenField-Org/Greenfield-Org.github.io
# Feature/136

### Steps to Test

***```npm install``` will need to be run to update your local with the new packages 

- run ```npm run storybook```
-  Navigate to component/Wave
- Verify the following conditions are met
- [ ] Any hex color can be used
- [ ] component can be animated or have no animation
- [ ] component can be used as a top border or a bottom border


* the wave is a special function created by us which does not live in Tailwind.  In order to keep what we have created the ```.css``` files was created.
